src/idd/idd.area.js

```diff
@@ -7,6 +7,8 @@ InteractiveDataDisplay.Area = function (div, master) {
     // Initialization (#1)
     var initializer = InteractiveDataDisplay.Utils.getDataSourceFunction(div, InteractiveDataDisplay.readCsv);
     var initialData = initializer(div);
+    var displayInLegend = $(div).attr("display-legend") == "false" ? false : true;
```

Comment:
@Andrey1024 Please use 'data-idd-legend-visible' to follow the styles of other idd attributes.